pkg/certs/cert-inspection/certgraphanalysis/metadata_options.go

```diff
@@ -199,6 +204,31 @@ func RewriteNodeIPs(nodeList []*corev1.Node) *metadataOptions {
 	}
 }
 
+func RewriteBootstrapIPs(bootstrapIP string) *metadataOptions {
+	return &metadataOptions{
+		rewriteSecretFn: func(secret *corev1.Secret) {
+			if secret.Namespace != "openshift-etcd" {
+				return
+			}
+
+			certHostNames, ok := secret.Annotations["auth.openshift.io/certificate-hostnames"]
```

Comment:
I suggest doing a replace of `descriptionWords[len(descriptionWords)-1]` on all secrets by first doing the list of etcd secrets and locating the name .  I'm willing to merge this as a starting point, but I'd like to see a refactor that identifies the bootstrap host name and replaces that.

Reply:
Right, I think its best to have `getBootstrapHostnameByIP` function which does that